packages/manager/apps/pci-billing/src/api/data/consumption.ts

```diff
@@ -18,103 +18,88 @@ export type TQuantity = {
   value: number;
 };
 
-export type TPeriod = { from: string; to: string };
+type TPeriod = { from: string; to: string };
 
-export type TCurrentUsage = {
-  hourlyUsage: {
-    rancher: {
-      reference: string;
-      region: string;
-      details: {
-        quantity: TQuantity;
-        totalPrice: {
-          currencyCode: string;
-          priceInUcents: number;
-          text: string;
-          value: number;
-        };
-      }[];
-    }[];
+type DetailedTotalPrice = {
+  currencyCode: string;
+  priceInUcents: number;
+  text: string;
+  value: number;
+};
+
+type ConsumptionItemDetail = {
+  quantity: TQuantity;
+  totalPrice: number;
+};
+
+type ConsumptionItemDetailPrice = {
+  quantity: TQuantity;
+  totalPrice: DetailedTotalPrice;
+};
+export type THourlyCurrentUsage = {
+  rancher: {
+    reference: string;
+    region: string;
+    details: ConsumptionItemDetailPrice[];
+  }[];
+  instance: {
+    details: {
+      instanceId: string;
+    } & ConsumptionItemDetail[];
+    quantity: TQuantity;
+    reference: string;
+    region: string;
+    totalPrice: number;
```

Comment:
You could use `ConsumptionItemDetail` type  intersection: 

```suggestion
    ConsumptionItemDetail & {
    reference: string;
    region: string;
```